demo/mcoa-demo/templates/policy-hub-grafana.yaml

```diff
@@ -0,0 +1,497 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: policy-hub-grafana
+  namespace: default
+spec:
+  disabled: false
+  policy-templates:
+  - objectDefinition:
+      apiVersion: policy.open-cluster-management.io/v1
+      kind: ConfigurationPolicy
+      metadata:
+        name: community-grafana
+      spec:
+        remediationAction: enforce
+        severity: high
+        pruneObjectBehavior: DeleteAll
+        object-templates:
+          - complianceType: musthave
+            objectDefinition:
+              apiVersion: v1
+              kind: Namespace
+              metadata:
+                name: grafana-operator
+          - complianceType: musthave
+            objectDefinition:
+              apiVersion: operators.coreos.com/v1
+              kind: OperatorGroup
+              metadata:
+                name: grafana-operator-kz2v9
+                namespace: grafana-operator
+                annotations:
+                  olm.providedAPIs: Grafana.v1beta1.grafana.integreatly.org,GrafanaDashboard.v1beta1.grafana.integreatly.org,GrafanaDatasource.v1beta1.grafana.integreatly.org,GrafanaFolder.v1beta1.grafana.integreatly.org
+              spec:
+                upgradeStrategy: Default
+          - complianceType: musthave
+            objectDefinition:
+              apiVersion: operators.coreos.com/v1alpha1
+              kind: Subscription
+              metadata:
+                name: grafana-operator
+                namespace: grafana-operator
+                labels:
+                  operators.coreos.com/grafana-operator.grafana-operator: ""
+              spec:
+                name: grafana-operator
+                channel: v5
+                installPlanApproval: Automatic
+                source: community-operators
+                sourceNamespace: openshift-marketplace
+                startingCSV: grafana-operator.v5.6.3
+  - extraDependencies:
+    - apiVersion: policy.open-cluster-management.io/v1
+      compliance: Compliant
+      kind: ConfigurationPolicy
+      name: community-grafana
+    - apiVersion: policy.open-cluster-management.io/v1
+      compliance: Compliant
+      kind: ConfigurationPolicy
+      name: signals-auth-copy
+    objectDefinition:
```

Comment:
Are the Grafana, GrafanaDatasource and GrafanaDashboard resources here really "extra dependencies"? I don't think they fit in this category. They are not dependencies, but in fact they are what the policy intends to deploy: a Grafana instance with set of datasources and dashboards. What's the reasoning behind moving them there as "extra dependencies"? 🤔

Reply:
Also, why is the ConfigurationPolicy that is deployed by this own Policy declared as "extra dependency"? This is kinda confusing to me.

Reply:
I think you are interpreting the other way around. What we are saying here is that for deploying the ConfigurationPolicy `grafana-instance` first we need to have the ConfigurationPolicy `community-grafana` and `signals-auth-copy` satisfied only then will we be able to apply the ConfigurationPolicy `grafana-instance`

Reply:
Note that the `objectDefinition` key is under the `extraDependencies` object, not outside.

Reply:
Ah, nevermind. I was reading the indentation wrong.